charmcraft.yaml

```diff
@@ -534,12 +534,8 @@ bases:
 parts:
   charm:
     plugin: charm
-    build-packages:
-    - git
-    charm-python-packages:
-    - setuptools==79.0.0
-    prime:
-    - templates/**
```

Comment:
Are you sure we want to remove the `templates` from the priming step? Am I missing something?

Reply:
from what i can tell, the charm plugin primed the templates folder.  But just for assurances i can leave it there

Reply:
Here's a list of [CHARM_OPTIONAL_FILES](https://github.com/canonical/charmcraft/blob/2.7.5/charmcraft/const.py#L60) handled by charmcraft including `templates`.  I couldn't find specifically where they are included in the repo but i do this in the log when building:

```
'prime': ['templates/**', 'src', 'venv', 'dispatch', 'hooks', 'templates', 'README.md', 'lib', 'LICENSE', 'lxd-profile.yaml']},
```

indicating both the `templates/**` and `templates` folks are included.  Let's leave it in -- since it's not necessary to remove